src/libs/ReportUtils.ts

```diff
@@ -5225,7 +5217,12 @@ function getIOUReportActionDisplayMessage(reportAction: OnyxEntry<ReportAction>)
         return Localize.translateLocal(translationKey, {amount: formattedAmount, payer: ''});
     }
 
-    const transaction = getTransaction(originalMessage.IOUTransactionID ?? '');
+    // This log to server is temporary and needed to determine if there is a case we need the transaction param
+    // when we call getIOUReportActionDisplayMessage from ReportActionItemMessage
+    if (shouldLog) {
+        Log.alert('Transaction Param Used when getIOUReportActionDisplayMessage was called from ReportActionItemMessage', {originalMessage});
```

Comment:
`originalMessage` shouldn't be logged because it will contain user's comments. Please only log the parts that are necessary for determining the code path like `paymentType`.

Reply:
@tgolen Logging will happen only when the report action is `IOU` type
https://github.com/Expensify/App/blob/65d9221ed7f1b269c1435aabfbe011ed1546478b/src/libs/ReportUtils.ts#L5191-L5192
 do you think there will be sensitive info in IOU type report actions ??

Reply:
Yeah, I would consider it sensitive. You could see who is requesting money from whom and for how much.

Reply:
Agreed. Only logging original message type.